AnselemOdims/iReporter
# push modified UI pages to gh-pages

### What does this PR do?
Modifies the UI 

#### Description of Task to be completed?
- Include view detail button in dashboard
- Reduce the font size
- Work on application responsiveness
- Align the profile page to be at the center
- Align the reports records header in the dashboard to be at the center
- On active, text colour should be stable
- Create view report details page
- Make boxes display inline block
- The status button should align center
- Add dummy pagination buttons
- Improve the view reports page design
- Design the edit reports page

#### How can it be manually tested
- Clone this repository
- cd into the directory
- Do git checkout chore-ui-and-endpoints-improvement-162497998
- Check UI improvements on github pages
